core/src/main/java/org/eclipse/hono/util/ResourceLimitsPeriod.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.hono.util;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The period definition corresponding to a resource limit for a tenant.
+ */
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class ResourceLimitsPeriod {
```

Comment:
this looks very similar to `DataVolumePeriod`, doesn't it?

Reply:
Yes. My idea is to deprecate `DataVolumePeriod` and use this in future for the `DataVolume`.

Reply:
sounds reasonable, the deprecation should then also be part of this PR, I guess

Reply:
yes, I will add it.